Makefile

```diff
@@ -17,7 +17,7 @@ code/compile:
 
 .PHONY: code/run
 code/run: code/gen
-	operator-sdk up local --namespace=$(NAMESPACE)
+	operator-sdk up local
```

Comment:
Is it required to remove the `--namespace` here? Is it better without it?

Reply:
--namespace string        The namespace where the operator watches for changes.
We want to watch multiple namespaces or am I wrong? (mdc, ups)?

Reply:
Ah yeah, sorry, I thought that flag had a different purpose. Sounds good! :+1: